pythonfmu/variables.py

```diff
@@ -68,6 +68,9 @@ def __init__(self, name: str, start: Optional[Any] = None, **kwargs):
         super().__init__(name, **kwargs)
         self.__attrs = {"start": start}
 
+    def __set_start__(self, value: float):
```

Comment:
```suggestion
    @start.setter
    def start(self, value: float):
```

in python, that is the syntax to define the setter of a property (see [documentation](https://docs.python.org/3/library/functions.html#property) for more info).

Then you can set the variable using:
```python
var.start = value
```